packages/editor/src/panels/assets/resources.tsx

```diff
@@ -205,7 +205,7 @@ export function FileCard({
             {name}
           </Text>
         </Tooltip>
-        <span className="text-xs text-[#375DAF]">{info}</span>
+        <span className="max-w-24 overflow-hidden text-ellipsis whitespace-nowrap text-xs text-[#375DAF]">{info}</span>
```

Comment:
In this case, using a hard width `w-24` would prob be more appropriate instead of `max-width` because otherwise, if the text is especially small, the misalignment issue would persist. This current code assumes the filename is always big/wide enough to force the span to be `w-24`

Reply:
makes sense. implemented!